utils/valcheck/attestation.go

```diff
@@ -13,12 +12,13 @@ type AttestationValueCheck struct {
 // Check returns error if value is invalid
 func (v *AttestationValueCheck) Check(value []byte) error {
```

Comment:
@nivBlox val check should be moved to operator IMO

Reply:
i think maybe it better to move it to go-client repo

Reply:
@nivBlox IMO we should move it to beacon package/go-client as its only referenced to it

Reply:
moved to beacon package